id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/database/IdentifiersDao.scala

```diff
@@ -47,7 +130,8 @@ class IdentifiersDao @Inject()(db: DB, identifiers: IdentifiersTable)
             .namedValues(
               identifiers.column.CanonicalID -> identifier.CanonicalID,
               identifiers.column.ontologyType -> identifier.ontologyType,
-              identifiers.column.MiroID -> identifier.MiroID
+              identifiers.column.MiroID -> identifier.MiroID,
+              identifiers.column.CalmAltRefNo -> identifier.CalmAltRefNo
```

Comment:
This change is untested for now, and only required so I can get tests working – I’ll add proper tests for this function in a separate patch.